doc/terminology.md

```diff
@@ -0,0 +1,106 @@
+# Terminology
+
+This pages gives an overview about the terms we use, when talking about interacting with VSS signals in-vehicle using KUKSA.val components.
```

Comment:
This document aims to describe all the terminology used in KUKSA.VAL, not just the terminology related to VSS (although that will be a large part of it).